src/index.html

```diff
@@ -2,15 +2,49 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div>
+      <div class="searchContainer" data-qa="big">
+        <form action="/" method="GET">
+          <img 
+          class="searchImage" 
+          src="./images/Search.svg" 
+          alt="glass"
+          >
```

Comment:
```suggestion
          <img 
            class="searchImage" 
            src="./images/Search.svg" 
            alt="glass"
          >
```
Форматування
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation